pkg/evaluation/save_results_flag.go

```diff
@@ -0,0 +1,36 @@
+package evaluation
+
+import (
+	"os"
+	"path/filepath"
+
+	"golang.org/x/sys/unix"
+)
+
+func IsWritableDirectory(filePath string) bool {
```

Comment:
If you indeed don't use this function, This whole file can be deleted as well, less code makes it simpler